From 4d4efce8ddd6baf237905566a408097ecb96c390 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?G=C3=BCnther=20Deschner?= <gd@samba.org>
Date: Wed, 27 Jan 2016 16:18:25 +0100
Subject: [PATCH 1058/1096] s3:smbXsrv.idl: add 8 byte channel_sequence number
 and request counters to IDL.

Guenther

Signed-off-by: Guenther Deschner <gd@samba.org>
Reviewed-by: Jeremy Allison <jra@samba.org>
(cherry picked from commit 31f33a3f3996a5fff9833540c8227600f4aa2a55)

BUG: https://bugzilla.samba.org/show_bug.cgi?id=11809
(cherry picked from commit 62669024caa30a676f661ea3f5e612f4d6a46bee)
---
 source3/librpc/idl/smbXsrv.idl | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/source3/librpc/idl/smbXsrv.idl b/source3/librpc/idl/smbXsrv.idl
index 4c6895a..1bfa51e 100644
--- a/source3/librpc/idl/smbXsrv.idl
+++ b/source3/librpc/idl/smbXsrv.idl
@@ -430,6 +430,7 @@ interface smbXsrv
 		uint32					durable_timeout_msec;
 		boolean8				durable;
 		DATA_BLOB				backend_cookie;
+		hyper					channel_sequence;
 	} smbXsrv_open_global0;
 
 	typedef union {
@@ -470,6 +471,8 @@ interface smbXsrv
 		[ignore] files_struct			*compat;
 		smbXsrv_open_flags			flags;
 		uint32					create_action;
+		hyper					request_count;
+		hyper					pre_request_count;
 	} smbXsrv_open;
 
 	typedef union {
-- 
2.9.0

